src/aof.c

```diff
@@ -218,7 +218,7 @@ void killAppendOnlyChild(void) {
     serverLog(LL_NOTICE,"Killing running AOF rewrite child: %ld",
         (long) server.child_pid);
     if (kill(server.child_pid,SIGUSR1) != -1) {
-        while(wait3(&statloc,0,NULL) != server.child_pid);
+        while(waitpid(-1, &statloc, 0) != server.child_pid);
```

Comment:
Do you think it would be better to follow the original coding rules(without space between param)?

Reply:
I think leaving a space between the parameters makes the code more readable and less crammed.

Reply:
It's the style of Redis that function calls don't have spaces between arguments(@madolson reminded me).
I used to be inclined to add spaces, just like you.

Reply:
I used to push on this, but no one does it consistently so I gave up.

Reply:
as far as i can tell, the majority of the function calls in redis **do** have spaces, although also many don't.
i wouldn't want to destroy the blame log by fixing that to be consistent, but if we already change that line, adding spaces seems like the right thing to me.